pipeline/metadata/5.3.yaml

```diff
@@ -1028,8 +1028,8 @@
     - rgw
     - stage-1
 
-- name: "test-empty-bucket-notification"
-  suite: "suites/pacific/rgw/tier-2_rgw_test-empty-bucket-notifcations.yaml"
+- name: "test-5.3-specific-fixes"
+  suite: "suites/pacific/rgw/tier-2_rgw_test_5.3_specific_fixes.yaml"
```

Comment:
Is there any specific reason that, suite: tier-2_rgw_test-empty-bucket-notifcations.yaml removed from pipeline execution?

Reply:
It is not removed, just the file has been renamed @anrao19

Reply:
the new file tier-2_rgw_test_5.3_specific_fixes.yaml is having 2 tests that are only fixed in rhcs-5.3 "test bucket notifcation with empty configuration" and  "test bi purge"

Reply:
@viduship, 
suite : tier-2_rgw_test_5.3_specific_fixes.yaml is a new file not a renamed suite of tier-2_rgw_test-empty-bucket-notifcations.yaml right? 
then Changes in file pipeline/metadata/5.3.yaml, looks like the existing suite tier-2_rgw_test-empty-bucket-notifcations.yaml removed from pipeline execution.

Reply:
Suite will be removed, but the test case is there in the newly named suite file

Reply:
IMO, If suite removed or renamed in same pr, that would have been more understandable

Reply:
Yeah, I agree. Consider it as part of the current activity of identifying suites for parallel execution.

Also, since this suite is not just specific to testing empty bucket notifications, hence the rationale is to rename it as it also has other tests.

Reply:
Shall we proceed to resolve it @anrao19 ?

Reply:
sure @viduship

Reply:
@anrao19 , I have removed the old suite file before renaming it.